include/SugarObjects/LanguageManager.php

```diff
@@ -202,6 +202,11 @@ public static function refreshLanguage($module, $lang, $loaded_mod_strings = arr
                     'custom/modules/'.$module.'/Ext/Language/'.$lang.'.lang.ext.php',
                  );
 
+        require_once('ModuleInstall/ModuleInstaller.php');
```

Comment:
Untrusted file inclusion:
```suggestion
        require_once __DIR__ . '/../../ModuleInstall/ModuleInstaller.php';
```